vcard2to3.py

```diff
@@ -223,11 +225,10 @@ def main(argv):
     replace = Replacer()
     if args.remove_dollar:
         replace.replace_filters.append(
-            (re.compile('^(N|FN):([^$]+)\$'), '\\1:\\2'))
+            (re.compile('^(N|FN):([^$]+)\\$'), '\\1:\\2'))
```

Comment:
I think this change is not correct. Double `\\` will match a literal `\`, but I want to match a literal `$`. The `$` has to be escaped with a single `\`, otherwise it would match the end of a line.

Reply:
Sorry, I was mistaken. Your change is correct.